src/kernels/raw/launcher/kernelEnvVarsService.node.ts

```diff
@@ -96,16 +88,16 @@ export class KernelEnvironmentVariablesService {
         if (interpreterEnv['PYTHONPATH']) {
             mergedVars['PYTHONPATH'] = interpreterEnv['PYTHONPATH'];
         }
-        otherEnvPathKey = Object.keys(customEditVars).find((k) => k.toLowerCase() == 'path');
-        if (otherEnvPathKey && customEditVars[otherEnvPathKey]) {
-            this.envVarsService.appendPath(mergedVars, customEditVars[otherEnvPathKey]!);
+        otherEnvPathKey = Object.keys(customEnvVars).find((k) => k.toLowerCase() == 'path');
+        if (otherEnvPathKey && customEnvVars[otherEnvPathKey]) {
+            this.envVarsService.appendPath(mergedVars, customEnvVars[otherEnvPathKey]!);
         }
         otherEnvPathKey = Object.keys(kernelEnv).find((k) => k.toLowerCase() == 'path');
         if (otherEnvPathKey && kernelEnv[otherEnvPathKey]) {
             this.envVarsService.appendPath(mergedVars, kernelEnv[otherEnvPathKey]!);
         }
-        if (customEditVars.PYTHONPATH) {
```

Comment:
renamed to make is clear that these are `custom Environment variables`. I am guessing the `Edit` was a typo from my end.